src/dotnet/CarbonAware.Plugin.Registration/Configuration/ServiceConfigurationExtension.cs

```diff
@@ -0,0 +1,26 @@
+using Microsoft.Extensions.DependencyInjection;
+using CarbonAware.Plugins.JsonReaderPlugin.Configuration;
+using CarbonAware.Plugins.CarbonIntensity.Configuration;
+
+namespace CarbonAware.Plugin.Configuration;
+public static class PluginServicesConfiguration
```

Comment:
Convention for extension methods is to generally name the static class after the thing its extending, so ServiceCollectionExtensions or IServiceCollectionExtensions would be a better name.